### PR TITLE
feat(suite-native): add enable/change/disable PIN protection e2e test

### PR DESCRIPTION
**Pull request by yanascz**
## Related Issues

#14615 #14616

### CODE REVIEW

File: suite-native/app/e2e/pageObjects/homeActions.ts [hunk 1]
```diff
@@ -7,6 +7,19 @@ class HomeActions {
             .withTimeout(10000);
     }
 
+    async waitForConnectingDeviceScreen() {
```
Comment (PeKne) [inline suite-native/app/e2e/pageObjects/homeActions.ts]: I'm not sure if this should be placed in this file. I believe that this file should contain only actions that are related specifically to the `HomeScreen`. `@screen/ConnectingDevice` is a different screen.

File: suite-native/app/e2e/pageObjects/homeActions.ts [hunk 2]
```diff
@@ -7,6 +7,19 @@ class HomeActions {
...
+        await waitFor(element(by.id('@screen/ConnectingDevice')))
+            .toBeVisible()
+            .withTimeout(10000);
+    }
+
+    async tapDeviceManagerSwitch() {
```
Comment (PeKne) [inline suite-native/app/e2e/pageObjects/homeActions.ts]: What about placing this to `deviceManagerActions`?

File: suite-native/app/e2e/tests/deviceSettings.test.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+        await onDeviceSettings.tapEnablePinProtectionButton();
+
+        await TrezorUserEnvLink.pressNo();
+        await onAlertSheet.tapPrimaryButton();
+
+        await TrezorUserEnvLink.pressYes();
+        await TrezorUserEnvLink.inputEmu('42');
+        await TrezorUserEnvLink.inputEmu('42');
+        await TrezorUserEnvLink.pressYes();
```
Comment (PeKne) [inline suite-native/app/e2e/tests/deviceSettings.test.ts:57]: There is not check if the action was successful. What about asserting if the success toast was displayed after each of these three pin actions?

Comment (PeKne) [reply suite-native/app/e2e/tests/deviceSettings.test.ts:57]: Or at least checking that redirect back to the device settings screen was successful after the last step of disabling the PIN again.

**Review by PeKne — APPROVED** [review]

File: suite-native/app/e2e/pageObjects/homeActions.ts (hunk 1 above)
Comment (yanascz) [reply suite-native/app/e2e/pageObjects/homeActions.ts]: Moved to a new file.

File: suite-native/app/e2e/pageObjects/homeActions.ts (hunk 2 above)
Comment (yanascz) [reply suite-native/app/e2e/pageObjects/homeActions.ts]: Makes sense.

File: suite-native/app/e2e/tests/deviceSettings.test.ts (hunk 3 above)
Comment (yanascz) [reply suite-native/app/e2e/tests/deviceSettings.test.ts:57]: Added `await onDeviceSettings.waitForScreen();`.

**Pull request merged**